src/views/results/html.ts

```diff
@@ -85,21 +86,21 @@ export function generateScroller(basicSelect: string, isCL: boolean): string {
                   isFetching = false;
                   noMoreRows = data.isDone;
 
-                  if (mustLoadHeaders && event.data.columnList) {
-                    setHeaders('resultset', event.data.columnList);
-                    mustLoadHeaders = false;
+                  if (needToInitializeTable && event.data.columnList) {
+                    initializeTable(event.data.columnList);
+                    needToInitializeTable = false;
                   }
 
                   if (data.rows && data.rows.length > 0) {
                     totalRows += data.rows.length;
-                    appendRows('resultset', data.rows);
+                    appendRows(data.rows);
                   }
 
-                  const nextButton = document.getElementById("nextButton");
                   if (data.rows === undefined && totalRows === 0) {
-                    nextButton.innerText = 'Query executed with no result set returned. Rows affected: ' + data.update_count;
+                    document.getElementById(messageSpanId).innerText = 'Statement executed with no result set returned. Rows affected: ' + data.update_count;
                   } else {
-                    nextButton.innerText = noMoreRows ? ('Loaded ' + totalRows + '. End of data') : ('Loaded ' + totalRows + '. Fetching more...');
+                    document.getElementById(statusId).innerText = noMoreRows ? ('Loaded ' + totalRows + '. End of data') : ('Loaded ' + totalRows + '. More available...');
```

Comment:
After trying it with `qsys2.columns`, it makes me debate the point of 'More available' going forward. Before you only saw it for a brief moment before it goes away again. I wonder if it should just say 'Loaded X' and 'Loaded X. End of data.'

Reply:
Not sure.  Without it there is no indication that there is more data, the user would just have to know that's the case because it _does not say **Done**_.

Reply:
Perhaps it is just the ellipsis that is not needed. Just 'More available.'